addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -273,12 +278,32 @@ private synchronized void refreshData() {
 	public void onConnectionLost() {
 		logger.info("Bridge connection lost. Updating thing status to OFFLINE.");
 		previousOnline = false;
-		updateStatus(ThingStatus.OFFLINE);
+		updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.COMMUNICATION_ERROR);
+		for (Thing thing : getThing().getThings()) {
+			ThingHandler handler = thing.getHandler();
+			if (handler != null && handler instanceof MaxDevicesHandler ) {
+				((MaxDevicesHandler) handler).setOffLine();
```

Comment:
> What I can do is to remove the thingstatus update to OFFLINE in this .setOffLine() if that helps

I'd suggest to do this change as the status will otherwise be set twice to OFFLINE.